IGCMenu/IGCMenu.h

```diff
@@ -39,7 +39,8 @@ typedef enum {
 @property int maxColumn;                            //Maximium number of column,default is 3
 @property int menuHeight;                           //height = width ,default is 65
 @property IGCMenuBackgroundOptions backgroundType;  //Default is BlurEffectDark
-
+@property (nonatomic) UIColor *menuItemColor;
```

Comment:
Property name seems little misguiding, it can be renamed to something `menuTextColor`